File: Android_Zone_Test/src/com/example/mylib_test/activity/animal/SVGActivity.java
package com.example.mylib_test.activity.animal;

import android.app.Activity;
import android.graphics.drawable.Animatable;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.ImageView;

import com.example.mylib_test.R;

/**
 * 绘制SVG
 * Created by LGL on 2016/4/16.
 */
public class SVGActivity extends Activity {

    private ImageView iv;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.a_svg);

        iv = (ImageView) findViewById(R.id.iv);
        iv.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                anim();
            }
        });
    }

    private void anim() {
        Drawable drawable = iv.getDrawable();
        if (drawable instanceof Animatable) {
            ((Animatable) drawable).start();
        }
    }

}

File: Android_Zone_Test/src/com/example/mylib_test/activity/animal/SVGActivity.kt
package com.example.mylib_test.activity.animal

import android.graphics.drawable.Animatable

import com.example.mylib_test.R
import com.zone.lib.base.controller.activity.BaseFeatureActivity
import kotlinx.android.synthetic.main.a_svg.*

/**
 * 绘制SVG
 * Created by LGL on 2016/4/16.
 */
class SVGActivity : BaseFeatureActivity() {

    override fun setContentView() {
        setContentView(R.layout.a_svg)
    }

    override fun initData() {
    }

    override fun setListener() {
        iv.setOnClickListener { anim() }
    }

    private fun anim() {
        val drawable = iv.drawable
        if (drawable is Animatable) {
            (drawable as Animatable).start()
        }
    }

}